content/self-host/rustdesk-server-pro/smtp/Microsoft365/_index.ro.md

```
@@ -22,7 +22,7 @@ Pentru configurarea SMTP generala, vezi [SMTP](../).
 | OAuth2 Client secret | `Value` al secretului creat in `Certificates & secrets` |
 
 Aceasta captura arata unde se introduc aceste valori in RustDesk:
-![](/docs/en/self-host/rustdesk-server-pro/smtp/Microsoft 365/images/21-RustDesk-SMTP-OAuth2-2.png)
+![](/docs/en/self-host/rustdesk-server-pro/smtp/Microsoft365/images/21-RustDesk-SMTP-OAuth2-2.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)